ios/XMTPModule.swift

```diff
@@ -70,20 +70,28 @@ extension Conversation {
 }
 
 public class XMTPModule: Module {
-    var apiEnvironments = [
-        "local": XMTP.ClientOptions.Api(
-            env: XMTP.XMTPEnvironment.local,
-            isSecure: false
-        ),
-        "dev": XMTP.ClientOptions.Api(
-            env: XMTP.XMTPEnvironment.dev,
-            isSecure: true
-        ),
-        "production": XMTP.ClientOptions.Api(
-            env: XMTP.XMTPEnvironment.production,
-            isSecure: true
-        ),
-    ]
+    private func apiEnvironments(env: String, appVersion: String?) -> XMTP.ClientOptions.Api {
+        switch env {
+            case "local":
+                return XMTP.ClientOptions.Api(
+                        env: XMTP.XMTPEnvironment.local,
+                        isSecure: false,
+                        appVersion: appVersion
+                    )
+            case "production":
+                return XMTP.ClientOptions.Api(
+                        env: XMTP.XMTPEnvironment.production,
+                        isSecure: false,
```

Comment:
I think this should be `isSecure: true` (for both `dev` and `prod`)

Reply:
Nice catch!